src/laddingpage/sections/Main.jsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from '../../components/button/Button'
```

Comment:
We are using .js and this project and not .jsx, pay attention on the desing pattern.